src/TabBar.js

```diff
@@ -25,7 +25,11 @@ class TabBar extends Component {
         `No action is defined for name=${el.props.name} ` +
         `actions: ${JSON.stringify(Object.keys(Actions))}`);
     }
-    Actions[el.props.name]();
+    if (el.props.onPress){
```

Comment:
Seems like a job for type checking with flow or typescript - not sure about going down the path of doing runtime type checking everywhere?  But cheers to anyone who would attempt passing a non-function to `onPress` :)